ignite-base/App/Services/RehydrationServices.js

```diff
@@ -1,16 +1,23 @@
 import ReduxPersist from '../Config/ReduxPersist'
 import { AsyncStorage } from 'react-native'
 import { persistStore } from 'redux-persist'
-import Reactotron from 'reactotron-react-native'
 
 const updateReducers = (store) => {
   const reducerVersion = ReduxPersist.reducerVersion
   const config = ReduxPersist.storeConfig
 
+
   // Check to ensure latest reducer version
   AsyncStorage.getItem('reducerVersion').then((localVersion) => {
     if (localVersion !== reducerVersion) {
-      Reactotron.log(`PURGING STORE ${localVersion} 'vs.' ${reducerVersion}`)
+      console.tron.display({
+        name: 'PURGE',
+        value: `Reducer Version Change
```

Comment:
Nothing wrong with what you've done here at all.   Props for doing this!

Another way would be to set `value` as an object.  That would have a nice table layout if you did.